pkg/limayaml/limayaml.go

```diff
@@ -7,20 +7,21 @@ import (
 )
 
 type LimaYAML struct {
-	Arch         Arch          `yaml:"arch,omitempty"`
-	Images       []File        `yaml:"images"` // REQUIRED
-	CPUs         int           `yaml:"cpus,omitempty"`
-	Memory       string        `yaml:"memory,omitempty"` // go-units.RAMInBytes
-	Disk         string        `yaml:"disk,omitempty"`   // go-units.RAMInBytes
-	Mounts       []Mount       `yaml:"mounts,omitempty"`
-	SSH          SSH           `yaml:"ssh,omitempty"` // REQUIRED (FIXME)
-	Firmware     Firmware      `yaml:"firmware,omitempty"`
-	Video        Video         `yaml:"video,omitempty"`
-	Provision    []Provision   `yaml:"provision,omitempty"`
-	Containerd   Containerd    `yaml:"containerd,omitempty"`
-	Probes       []Probe       `yaml:"probes,omitempty"`
-	PortForwards []PortForward `yaml:"portForwards,omitempty"`
-	Network      Network       `yaml:"network,omitempty"`
+	Arch                 Arch              `yaml:"arch,omitempty"`
+	Images               []File            `yaml:"images"` // REQUIRED
+	CPUs                 int               `yaml:"cpus,omitempty"`
+	Memory               string            `yaml:"memory,omitempty"` // go-units.RAMInBytes
+	Disk                 string            `yaml:"disk,omitempty"`   // go-units.RAMInBytes
+	Mounts               []Mount           `yaml:"mounts,omitempty"`
+	SSH                  SSH               `yaml:"ssh,omitempty"` // REQUIRED (FIXME)
+	Firmware             Firmware          `yaml:"firmware,omitempty"`
+	Video                Video             `yaml:"video,omitempty"`
+	Provision            []Provision       `yaml:"provision,omitempty"`
+	Containerd           Containerd        `yaml:"containerd,omitempty"`
+	Probes               []Probe           `yaml:"probes,omitempty"`
+	PortForwards         []PortForward     `yaml:"portForwards,omitempty"`
+	Network              Network           `yaml:"network,omitempty"`
+	EnvironmentVariables map[string]string `yaml:"environmentVariables,omitempty"`
```

Comment:
I think we can just call it "Env".
Also, the type should be probably `map[string]*string`

Reply:
> Also, the type should be probably `map[string]*string`

@AkihiroSuda Why did you ask for the type to be changed? I noticed that in template expansion you cannot pass a `*string` to e.g. `{{ printf "%s=%q" $var $key }}`, you can only interpolate directly, i.e. `{{$key}}` (or at least that's how I remember it when I last tried it; I may have made some other mistake).

Just asking about this for general understanding; we don't actually need to pass the values to `printf`.